packages/design-tokens/less/variable-identifiers.less

```diff
@@ -46,6 +46,22 @@
 @kz-var-id-color-wisteria-600-rgb-params: --kz-var-color-wisteria-600-rgb-params;
 @kz-var-id-color-wisteria-700-rgb-params: --kz-var-color-wisteria-700-rgb-params;
 @kz-var-id-color-wisteria-800-rgb-params: --kz-var-color-wisteria-800-rgb-params;
+@kz-var-id-color-purple-100: --kz-var-color-purple-100;
```

Comment:
[My understanding](https://github.com/cultureamp/kaizen-design-system/pull/1741/files#pullrequestreview-712288674) from this morning is these variable identifiers would become `$color-<colorName>-<scale>-id`?

Reply:
the new colors probably shouldn't be in variable identifiers i think